packages/select/theme/lumo/vaadin-select-styles.js

```diff
@@ -7,42 +7,42 @@ import { registerStyles, css } from '@vaadin/vaadin-themable-mixin/register-styl
 import '@vaadin/vaadin-lumo-styles/sizing.js';
 import '@vaadin/vaadin-lumo-styles/style.js';
 import '@vaadin/vaadin-lumo-styles/font-icons.js';
-import '@vaadin/vaadin-lumo-styles/mixins/menu-overlay.js';
-import '@vaadin/text-field/theme/lumo/vaadin-input-field-shared-styles.js';
-
-registerStyles(
-  'vaadin-select',
-  css`
-    :host(:not([theme*='align'])) ::slotted([slot='value']) {
-      text-align: start;
-    }
-
-    [part='input-field'] ::slotted([slot='value']) {
-      font-weight: 500;
-    }
-
-    /* placeholder styles */
-    :host(:not([has-value])) [part='input-field'] ::slotted([slot='value']) {
-      color: inherit;
-      transition: opacity 0.175s 0.1s;
-      opacity: 0.5;
-    }
-
-    :host([has-value]) [part='input-field'] ::slotted([slot='value']) {
-      color: var(--lumo-body-text-color);
-    }
-
-    [part='toggle-button']::before {
-      content: var(--lumo-icons-dropdown);
-    }
-
-    /* Highlight the toggle button when hovering over the entire component */
-    :host(:hover:not([readonly]):not([disabled])) [part='toggle-button'] {
-      color: var(--lumo-contrast-80pct);
-    }
-  `,
-  { moduleId: 'lumo-select', include: ['lumo-input-field-shared-styles'] }
-);
+import { menuOverlay } from '@vaadin/vaadin-lumo-styles/mixins/menu-overlay.js';
+import { inputFieldShared } from '@vaadin/text-field/theme/lumo/vaadin-input-field-shared-styles.js';
+
+const select = css`
+  :host(:not([theme*='align'])) ::slotted([slot='value']) {
+    text-align: start;
+  }
+
+  [part='input-field'] ::slotted([slot='value']) {
+    font-weight: 500;
+  }
+
+  /* placeholder styles */
+  :host(:not([has-value])) [part='input-field'] ::slotted([slot='value']) {
+    color: inherit;
+    transition: opacity 0.175s 0.1s;
+    opacity: 0.5;
+  }
+
+  :host([has-value]) [part='input-field'] ::slotted([slot='value']) {
+    color: var(--lumo-body-text-color);
+  }
+
+  [part='toggle-button']::before {
+    content: var(--lumo-icons-dropdown);
+  }
+
+  /* Highlight the toggle button when hovering over the entire component */
+  :host(:hover:not([readonly]):not([disabled])) [part='toggle-button'] {
+    color: var(--lumo-contrast-80pct);
+  }
+`;
+
+registerStyles('vaadin-select', [...inputFieldShared, select], {
+  moduleId: 'lumo-select'
+});
```

Comment:
Same as above, I think this can be changed to use single line.

Reply:
Done.